clex/driver.c

```diff
@@ -400,7 +400,13 @@ void reverse_toks (int idx)
   int matched = 0;
   int which = 0;
   int i;
+
+#ifdef _MSC_VER
+  int *saved = calloc(N, sizeof(int));
```

Comment:
Yes, sadly variable length arrays are not supported (https://msdn.microsoft.com/en-us/library/zb1574zs.aspx).
Moreover, _alloca is marked as deprecated (https://msdn.microsoft.com/en-us/library/wb1s57t5.aspx) and the alternative _malloca does again require calls to _freea (https://msdn.microsoft.com/en-us/library/5471dc8s.aspx).
It might be a bit more efficient to use _malloca as the stack is used in *most* cases. Shall I change the calls from ```calloc``` to ```_malloca```?